### DISCUSSION TITLE
app ideas that could be good in the script

### DISCUSSION THREAD

**Discussion by DadyRick** [ChrisTitusTech/winutil#2493] [Ideas]
apps i use and maybe useful to others
1. filebot - pulls from common data bases like imdb and works well with anime and has plex Intergration to help with plex's data scanning thing to add stuff
2. IDM (internet download manager) it acks very much like a torrent program, but I'm herring story's that i shouldn't use it because the app is a virus but I've being using for almost 2 years now and i haven't had any trouble with it and its made download big file so muck easier and makes download not crash on me and its a plus that the browser extension allows downloading imbedded video from sites like YouTube and works with with other programs that help with downloading vods from twitch so i can do my editing.
3. Twitch Downloader (https://github.com/lay295/TwitchDownloader)
4. PURPLE (NCSOFT game launcher) for Blade and Sorcery and Aion [ i wish they where on steam ]
5. DaVinci Resolve (video editor)
6. Riot games launcher

it would be amazing to have these in the script if its a no all well

**Comment by Xenod666** [comment]
id like to add Blizzard Launcher too many of their game are played for many of us. 

It would be very awsome.

**Reply by DadyRick** [reply]
Isn’t all blizzard games on steam now because of Microsoft buying them out

**Comment by cerez11** [comment]
adding [irfanview](https://www.irfanview.com/) would be nice. 
It's one of the best image-viewers out there

**Reply by GrafDogi** [reply]
Yes please! I am using it for years! :)

**Comment by Xenod666** [comment]
No, actually Wow is not on steam and many other too.

**Comment by cde2496** [comment]
I believe those apps are quite useful please add to WinUtil:

1. Coodesker https://www.coodesker.com/
2. DirectoryOpus https://www.gpsoft.com.au/
3. GameSaveManager https://www.gamesave-manager.com/
4. Java https://www.java.com/en/
5. MKVToolNix https://www.bunkus.org/videotools/mkvtoolnix/
6. MPC-BE https://sourceforge.net/projects/mpcbe/
7. SIV http://rh-software.com/
8. StartAllBack https://www.startallback.com/
9. SyncBackFree https://www.2brightsparks.com/download-syncbackfree.html
10. Volume² https://irzyxa.blogspot.com/
11. W10Privacy https://www.winprivacy.de/english-home/

**Comment by xSalty1x** [comment]
Biniware Run would be a great add https://www.biniware.com

**Comment by xSalty1x** [comment]
Biniware Run is a legend of Application Launcher and I nominate it inclusion. Light, fast, free, easy to configure and has an excellent dark mode.

AntiMicroX  Which is a great cross os controller mapper or most any device mapper.  Simple, light and works like a trick.

Lossless-Scaling Which is a program you can use to use your GPU to implement scaling, frame generation and lot's of other things in games but in any program really.  Very cool if you want a game that is running at 60 fps to run at 120 fps for example with no loss of visual quality.

https://biniware.com
https://github.com/AntiMicroX/antimicrox/releases
https://github.com/Gamepagge/Lossless-Scaling-Download/blob/main/README.md
